### DISCUSSION TITLE
Is it synchronous parallel development and compilation?

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win-AVX2#60] [General]
110.0 special edition and M111.0windows10&11 version, and because of 110.0 special edition, it needs to modify the runtime and dynamic link library (a series of dll files) a lot, so the new version is difficult to produce and the progress is slow? Then can't, let users of Windows 10 and 11 first experience Th build M111.0 first? ; Or directly follow up M112.0 in the middle of this month, after all, there are a lot of version spans involved in this release of thorium.

**Comment by gz83** [comment]
> 110.0 special edition and M111.0windows10&11 version, and because of 110.0 special edition, it needs to modify the runtime and dynamic link library (a series of dll files) a lot, so the new version is difficult to produce and the progress is slow? Then can't, let users of Windows 10 and 11 first experience Th build M111.0 first? ; Or directly follow up M112.0 in the middle of this month, after all, there are a lot of version spans involved in this release of thorium.

@whxuezhe Thorium recently intends to provide a special version for Win7/8/8.1 systems. The M110 version and newer versions released later do not support Win7/8/8.1 systems. Thorium will not provide this version for users of these systems.

Thorium for Win10/11 systems will continue to be updated, and this project has not been abandoned. I and other members of the Thorium project team know that a new version needs to be released as soon as possible. Based on the above, please wait patiently for the release of the new version, please **DO NOT** post frequently about updating the Thorium browser.
